charts/argo-workflows/templates/crds.yaml

```diff
@@ -0,0 +1,14 @@
+{{- if .Values.crds.install }}
+{{- $pattern := ternary "files/crds/full/*.yaml" "files/crds/minimal/*.yaml" .Values.crds.full }}
+{{- range $path, $_ :=  .Files.Glob  $pattern }}
+{{- if not (contains "clusterworkflowtemplates" $path) }}
+---
+{{ $.Files.Get $path }}
+{{- end }}
+{{- end }}
+{{- if or (.Values.server.clusterWorkflowTemplates.enabled) (.Values.controller.clusterWorkflowTemplates.enabled) }}
```

Comment:
I would suggest putting this condition into line 4 so that you don't have to complicate the code with two loops:
```
{{- if or (contains "clusterworkflowtemplates" $path | not) .Values.server.clusterWorkflowTemplates.enabled .Values.controller.clusterWorkflowTemplates.enabled }}
```

Reply:
Done